README.md

```diff
@@ -62,7 +61,7 @@
 - [Start with Docker](docker/README.md)
 - [Start with Kubernetes](helm/README.md)
 
-Click [Document](https://streampark.apache.org/docs/user-guide/quick-start) for more information
+Click [Official documentation](https://streampark.apache.org/docs/user-guide/quick-start) for more information
```

Comment:
I think this should be consistent, why is one using documentation and the other is official documentation?